RetailCoder.VBE/UnitTesting/IAssert.cs

```diff
@@ -10,40 +10,40 @@ public interface IAssert
         /// </summary>
         /// <param name="condition">Any Boolean value or expression.</param>
         /// <param name="message">An optional message to display if the assertion fails.</param>
-        void IsTrue(bool condition, string message = null);
+        void IsTrue(bool condition, string message = "");
```

Comment:
I deliberately had those as `null` because `String.Empty` wouldn't compile. Makes no difference really... except for my OCD 😉